docs/approvals-service.md

```diff
@@ -8,33 +8,36 @@ Helper functions for handling form submission approvals
 import { approvalsService } from '@oneblink/apps'
 ```
 
-- [`FormApprovalFlowStep`](#formapprovalflowstep)
+- [`FormApprovalFlowInstanceStep`](#formapprovalflowinstancestep)
 - [`FormApprovalFlowInstance`](#formapprovalflowinstance)
 - [`FormSubmissionApproval`](#formsubmissionapproval)
 - [`getFormSubmissionApprovals()`](#getformsubmissionapprovals)
 - [`getFormSubmissionApproval()`](#getformsubmissionapproval)
 - [`updateFormSubmissionApproval()`](#updateformsubmissionapproval)
 - [`retrieveFormSubmissionApprovalSubmission()`](#retrieveformsubmissionapprovalsubmission)
 
-### FormApprovalFlowStep
+### FormApprovalFlowInstanceStep
 
-| Property   | Type     | Description                                                              |
-| ---------- | -------- | ------------------------------------------------------------------------ |
-| `type`     | `string` | `'SINGLE'`                                                               |
-| `label`    | `string` | The unique label for the step                                            |
-| `username` | `string` | The username of the user that will be assigned an approval for this step |
+| Property                           | Type                                  | Description                                                              |
+| ---------------------------------- | ------------------------------------- | ------------------------------------------------------------------------ |
+| `label`                            | `string`                              | The unique label for the step                                            |
+| `group`                            | `string`                              | The username of the user that will be assigned an approval for this step |
+| `isConditional`                    | `boolean \| undefined`                | Indicates if the step is only required if a specfied condition is met    |
+| `requiresAllConditionalPredicates` | `boolean \| undefined`                | Indicates if if all conditions must be met or only one                   |
+| `conditionalPredicates`            | `ConditionalPredicate[] \| undefined` | Array of conditions                                                      |
```

Comment:
`isConditional`, `requiresAllConditionalPredicates` and `conditionalPredicates` will not be on this type.